healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc

```diff
@@ -2,11 +2,11 @@
 
 The following are a collection of assets to support MuleSoft customers who are developing Fast Healthcare Interoperability Resources (FHIR) APIs using MuleSoft. Visit the https://www.hl7.org/fhir/[official HL7 website] for more information.
 
-* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/hls-fhirjson-to-raml-sys-api-spec/[FHIR RAML auto-generator] - Utility to generate RAML specification(s) for FHIR resources directly from the JSON schema provided by HL7.
-* xref:fhir-r4-us-core-profiles.adoc[FHIR R4 US Core Profile Assets]
-* xref:fhir-r4-resources.adoc[FHIR R4 Resources List]
-* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/fhir-error-policy/[FHIR Error API Policy] - Captures error responses and formats them in an FHIR OperationOutcome-compliant format, since Anypoint Platform's out-of-the-box (OOTB) policies do not provide error messages in an FHIR-compliant format. This policy can also be used with other OOTB API policies provided in Anypoint Platform, such as OAuth or JWT.
-* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/hc-accelerator-okta-smartfhir/[Okta SMART on FHIR^] - Implementation template that walks you through how to set up a reference SMART on FHIR implementation on Anypoint Platform using Okta as the identity provider.
+* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/hls-fhirjson-to-raml-sys-api-spec/[FHIR RAML auto-generator] - Utility to generate RAML specifications for FHIR resources from the JSON schema provided by HL7.
+* xref:fhir-r4-us-core-profiles.adoc[FHIR R4 US Core Profile Assets] - A collection of API specifications, data types, and libraries designed to meet FHIR USCDI Core Profile compliance standards.
+* xref:fhir-r4-resources.adoc[FHIR R4 Resources List] - A collection of API specifications, data types, and libraries designed to meet FHIR R4 compliance standards.
+* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/fhir-error-policy/[FHIR Error API Policy] - A custom policy that captures error responses and formats them into `OperationOutcome`, which is a FHIR compliant format.
```

Comment:
@ppalu what do you think of this phrasing? Seems a little off to me but you would know better if this accurately captures what we had built